src/webserver/WebServer.jl

```diff
@@ -130,6 +130,7 @@ function run(session::ServerSession, pluto_router)
     hostIP = parse(Sockets.IPAddr, host)
     if port === nothing
         port, serversocket = Sockets.listenany(hostIP, UInt16(1234))
+        session.options.server.port = port
```

Comment:
I added this because otherwise server-side event consumers would have no way of knowing what port Pluto is running on. This would only assign the configured port when it is not explicitly provided.